src/resource.ts

```diff
@@ -23,22 +24,22 @@ export const resource = <T, U = t.mixed, C = any, TO = T, UO = U>(
   return (options: any) => (req: Request) => {
     const durationStart = Date.now();
     return Promise.resolve(desc.init(options))
-      .then(context => desc.authorized(req.properties.headers, context))
-      .then(context => desc.exists(req.properties.headers, context))
-      .then(context => desc.forbidden(req.properties.headers, context))
-      .then(context =>
-        decode(desc.type[0], req.body).then(data =>
+      .then((context) => desc.authorized(req.properties.headers, context))
+      .then((context) => desc.exists(req.properties.headers, context))
+      .then((context) => desc.forbidden(req.properties.headers, context))
+      .then((context) =>
+        decode(desc.type[0], req.body).then((data) =>
           Promise.resolve(desc.update(data, context))
-            .then(_context => desc.response(_context))
-            .then(result => decode(desc.type[1], result))
+            .then((_context) => desc.response(_context))
+            .then((result) => decode(desc.type[1], result))
             .then(response(req))
         )
       )
-      .then(success => {
+      .then((success) => {
         _logger.info(
           createDurationLogInfo(req, "Response sent", durationStart, Date.now())
         );
         return success;
-      }, errorHandler(req, _logger));
+      }, errorHandler(req, _logger, desc.defaultNackDelayMs));
```

Comment:
I haven’t had time to look through it thoroughly, but are we nacking request-messages (not just event messages), with a delay?
A more appropriate behaviour in an rpc scenario is to respond with the error and then ack so the request disappears. It's up to the requester to retry, or not.

Reply:
Yes, that’s more appropriate. But mq has always nacked if `err instanceof Error`, even for RPC.

Reply:
Ok, I don't see a problem in changing it but would probably be good with a second opinion. We may have use cases where the requester simply doesn't retry...
If we do change it, I'm wondering whether we should have two different errorHandlers.

Reply:
Might be good but requires more thought and needs testing with out system. Let's not do it in this PR

Reply:
I agree, let’s not do it in this PR. Any other comments?